src/lib/components/search-attribute-filter/index.svelte

```diff
@@ -209,6 +217,14 @@
         {/if}
       </div>
     {/if}
+    {#if showFilter}
+      <CopyButton
```

Comment:
We'll also need to take into account that `SearchAttributeFilter` is used on the `/schedules` page as well ⬇️ 

![Screenshot 2025-03-18 at 5 08 41 PM](https://github.com/user-attachments/assets/5d954f9c-c943-4888-adf6-5574daef9596)

Reply:
Thanks for noticing this Laura. I should have searched the repo for where this was imported. I appreciate you seeing this.